### DISCUSSION TITLE
[Anisotropy] GBAnisotropy Question when there are many grains

### DISCUSSION THREAD

**Discussion by syeonk** [idaholab/moose#18091] [Q&A Modules: Phase field] [answered]
Hello!

I'm interested in grain growth mechanism, so I have referred your simulation nowadays.
I'm really happy to use phase field simulation easily :)

But I have a problem on the anisotropy grain growth.
When I run it through below conditions, it did not converge at the time step although I apply large number of iterations in executioner. Moreover, it did not look like general grain growth case. (boundary goes to bend)
(Conditions: in 'combined' module, set initial points of 22 grains using txt file.
                    100X100 meshes on 2D surface (10nmX10nm).
                    time step 0.25 ns.
                     making matrices of GB properties (GB energy, mobility prefactor) through a paper, 'Eisuke Miyoshi, et al. Large-scale phase-field study of anisotropic grain growth: Effects of misorientation-dependent grain boundary energy and mobility, Comp. Mater. Sci. 186 (2021) 109992')

I also tested using 'combined' module in moose program for 6 grains on same conditions (just reduce the number of grains), however, it works.

Is there a problem when I increase the number of grains?

I would appreciate your reply :)
Thanks,

**Comment by simopier** [comment, marked as answer]
Hi @syeonk, 

Could you provide the input files and other txt files of the two simulations (the one that works and the one that does not), and the associated output files? That would help identify where the issue is coming from. 

Thank you!
